src/components/ArticlesList.jsx

```diff
@@ -0,0 +1,42 @@
+import {useState, useEffect} from "react";
+import { getArticles } from "../utils/api";
+import ArticleCard from "./ArticleCard";
+
+
+const ArticlesList = () => {
+const [isLoading, setIsLoading] = useState(true);
+const [articles, setArticles] = useState([])
+
+useEffect(() => {
+    setIsLoading(true);
+    getArticles().then(({articles}) => {
+        setArticles(articles)
+        setIsLoading(false)
+    })
+}, []);
+
+
+if (isLoading) {
+    return <p className="Loading">Loading...</p>
+} 
+
+return (
+    <div className="Article_List">
+        {articles.map((article) => {
+            
+            return (
+                <ArticleCard
```

Comment:
As this is a list of articles the map should be wrapped in a `ul` or `ol` tag. The `ArticleCard` should also return an `li` instead of a `div`.